modules/openapi-generator/src/main/resources/JavaJaxRS/cxf-cdi/RestApplication.mustache

```diff
@@ -3,7 +3,7 @@ package {{invokerPackage}};
 import javax.ws.rs.ApplicationPath;
 import javax.ws.rs.core.Application;
 
-@ApplicationPath("/")
+@ApplicationPath("{{contextPath}}")
```

Comment:
@michbeck100 I would suggest using `{{{contentPath}}}` instead to avoid special characters (e.g. @) being escaped.

Reply:
I mean `{{{contextPath}}}`. Sorry for the typo. (fixed above)